src/js/application/plugins.js

```diff
@@ -361,6 +354,26 @@ export const semanticui = {
           };
           value(callback);
         } else {
+          if (key === "modal") {
+            // We do something a fun here and move the element into ``body``
```

Comment:
"a fun" is correct or is a typo?

```suggestion
            // We do something fun here and move the element into ``body``
```

Reply:
More than fun is a hack to me 😅 

```suggestion
            // HACK: We do something fun here and move the element into ``body``
```